src/environmentd/tests/auth.rs

```diff
@@ -988,7 +989,7 @@ fn test_auth() -> Result<(), Box<dyn Error>> {
                 assert: Assert::Success,
             },
             TestCase::Http {
-                user: "mz_system",
```

Comment:
Needs a test for logging in as mz_system but failing.

Reply:
Added